testing/endpoint_multihop.go

```diff
@@ -288,6 +331,86 @@ func (ep *EndpointM) ProofHeight() clienttypes.Height {
 	return ep.GetClientState().GetLatestHeight().(clienttypes.Height)
 }
 
+// USED FOR TESTING ONLY!!!
```

Comment:
this data structure is only for testing (under the testing dir) so no need to call it out